javaslang/src/test/java/javaslang/collection/euler/Euler17Test.java

```diff
@@ -99,7 +99,7 @@ default int letterCount(Seq<Integer> range) {
             1_000, "thousand",
             1_000_000, "million",
             1_000_000_000, "billion"
-    );
+    ).grouped(2).toSortedMap(pair -> Tuple.of((Integer) pair.get(0), (String) pair.get(1)));
```

Comment:
@paplorinc is also a fox - just saying that we have Java-trickery experts here :)